scripts/dev/gen_versioninfo.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python3
+# vim: ft=python fileencoding=utf-8 sts=4 sw=4 et:
+
+# Copyright 2018 Florian Bruhin (The Compiler) <mail@qutebrowser.org>
+#
+# This file is part of qutebrowser.
+#
+# qutebrowser is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# qutebrowser is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with qutebrowser.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Generate file_version_info.txt for Pyinstaller use with Windows builds."""
+
+import os.path
+import sys
+
+from PyInstaller.utils.win32 import versioninfo as vs
+
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), os.pardir,
+                                os.pardir))
+
+import qutebrowser
+
+out_filename = 'misc/file_version_info.txt'
```

Comment:
Please also call `utils.change_cwd()` before writing the file to make sure it ends up at the right place - see e.g. `build_release.py` for an example.